From 5bc517c50664549a1cf8d057b4bc670731efb978 Mon Sep 17 00:00:00 2001
From: Jonathan Neal <jonathantneal@hotmail.com>
Date: Mon, 8 Jul 2019 21:04:46 -0400
Subject: [PATCH 073/103] 6.7.0

* Update tests
* Update browsers option to populate overrideBrowserslist in autoprefixer
* Safely update dependencies (minor and patch)
---
 CHANGELOG.md                 | 12 ++++++++++
 package.json                 | 30 +++++++++++------------
 src/postcss.js               |  2 +-
 test/basic.expect.css        | 46 ++++++++++++++++--------------------
 test/basic.stage0.expect.css | 46 ++++++++++++++++--------------------
 5 files changed, 70 insertions(+), 66 deletions(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index f7b3147..05a6190 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,17 @@
 # Changes to PostCSS Preset Env
 
+### 6.7.0 (July 8, 2019)
+
+- Fixed the issue of autoprefixer alerting an upcoming change to the API
+- Updated `autoprefixer` to 9.6.1 (minor)
+- Updated `browserslist` to 4.6.4 (minor)
+- Updated `cssdb` to 4.4.0 (minor)
+- Updated `caniuse-lite` to 1.0.30000981 (patch)
+- Updated `postcss` to 7.0.17 (patch)
+- Updated `postcss-color-hex-alpha` to 5.0.3 (patch)
+- Updated `postcss-custom-media` to 7.0.8 (patch)
+- Updated `postcss-custom-properties` to 8.0.11 (patch)
+
 ### 6.6.0 (February 28, 2019)
 
 - Moved browserslist detection from using each input file per process to using
diff --git a/package.json b/package.json
index 3ffdb6c..9f72176 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "postcss-preset-env",
-  "version": "6.6.0",
+  "version": "6.7.0",
   "description": "Convert modern CSS into something browsers understand",
   "author": "Jonathan Neal <jonathantneal@hotmail.com>",
   "license": "CC0-1.0",
@@ -27,22 +27,22 @@
     "node": ">=6.0.0"
   },
   "dependencies": {
-    "autoprefixer": "^9.4.9",
-    "browserslist": "^4.4.2",
-    "caniuse-lite": "^1.0.30000939",
+    "autoprefixer": "^9.6.1",
+    "browserslist": "^4.6.4",
+    "caniuse-lite": "^1.0.30000981",
     "css-blank-pseudo": "^0.1.4",
     "css-has-pseudo": "^0.10.0",
     "css-prefers-color-scheme": "^3.1.1",
-    "cssdb": "^4.3.0",
-    "postcss": "^7.0.14",
+    "cssdb": "^4.4.0",
+    "postcss": "^7.0.17",
     "postcss-attribute-case-insensitive": "^4.0.1",
     "postcss-color-functional-notation": "^2.0.1",
     "postcss-color-gray": "^5.0.0",
-    "postcss-color-hex-alpha": "^5.0.2",
+    "postcss-color-hex-alpha": "^5.0.3",
     "postcss-color-mod-function": "^3.0.3",
     "postcss-color-rebeccapurple": "^4.0.1",
-    "postcss-custom-media": "^7.0.7",
-    "postcss-custom-properties": "^8.0.9",
+    "postcss-custom-media": "^7.0.8",
+    "postcss-custom-properties": "^8.0.11",
     "postcss-custom-selectors": "^5.1.2",
     "postcss-dir-pseudo-class": "^5.0.0",
     "postcss-double-position-gradients": "^1.0.0",
@@ -66,16 +66,16 @@
     "postcss-selector-not": "^4.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
-    "babel-eslint": "^10.0.1",
-    "eslint": "^5.14.1",
+    "@babel/core": "^7.5.0",
+    "@babel/preset-env": "^7.5.2",
+    "babel-eslint": "^10.0.2",
+    "eslint": "^5.16.0",
     "eslint-config-dev": "^2.0.0",
     "postcss-simple-vars": "^5.0.2",
     "postcss-tape": "^4.0.0",
     "pre-commit": "^1.2.2",
-    "rollup": "^1.3.2",
-    "rollup-plugin-babel": "^4.3.2"
+    "rollup": "^1.16.6",
+    "rollup-plugin-babel": "^4.3.3"
   },
   "eslintConfig": {
     "extends": "dev",
diff --git a/src/postcss.js b/src/postcss.js
index 39d22b2..322cd8c 100644
--- a/src/postcss.js
+++ b/src/postcss.js
@@ -23,7 +23,7 @@ export default postcss.plugin('postcss-preset-env', opts => {
 	const sharedOpts = initializeSharedOpts(Object(opts));
 	const stagedAutoprefixer = autoprefixerOptions === false
 		? () => {}
-	: autoprefixer(Object.assign({ browsers }, autoprefixerOptions));
+	: autoprefixer(Object.assign({ overrideBrowserslist: browsers }, autoprefixerOptions));
 
 	// polyfillable features (those with an available postcss plugin)
 	const polyfillableFeatures = cssdb.concat(
diff --git a/test/basic.expect.css b/test/basic.expect.css
index 69469ed..ce6715c 100644
--- a/test/basic.expect.css
+++ b/test/basic.expect.css
@@ -139,21 +139,21 @@
 	clear: none;
 	clip: auto;
 	color: #000;
-	-webkit-columns: auto;
-	        columns: auto;
-	-webkit-column-count: auto;
-	        column-count: auto;
-	-webkit-column-fill: balance;
-	        column-fill: balance;
+	-moz-columns: auto;
+	     columns: auto;
+	-moz-column-count: auto;
+	     column-count: auto;
+	-moz-column-fill: balance;
+	     column-fill: balance;
 	grid-column-gap: normal;
-	-webkit-column-gap: normal;
-	        column-gap: normal;
-	-webkit-column-rule: medium none currentColor;
-	        column-rule: medium none currentColor;
-	-webkit-column-span: 1;
-	        column-span: 1;
-	-webkit-column-width: auto;
-	        column-width: auto;
+	-moz-column-gap: normal;
+	     column-gap: normal;
+	-moz-column-rule: medium none currentColor;
+	     column-rule: medium none currentColor;
+	-moz-column-span: 1;
+	     column-span: 1;
+	-moz-column-width: auto;
+	     column-width: auto;
 	content: normal;
 	counter-increment: none;
 	counter-reset: none;
@@ -193,10 +193,8 @@
 	page-break-after: auto;
 	page-break-before: auto;
 	page-break-inside: auto;
-	-webkit-perspective: none;
-	        perspective: none;
-	-webkit-perspective-origin: 50% 50%;
-	        perspective-origin: 50% 50%;
+	perspective: none;
+	perspective-origin: 50% 50%;
 	position: static;
 	right: auto;
 	-moz-tab-size: 8;
@@ -204,18 +202,16 @@
 	     tab-size: 8;
 	table-layout: auto;
 	text-align: left;
-	text-align-last: auto;
+	-moz-text-align-last: auto;
+	     text-align-last: auto;
 	text-decoration: none;
 	text-indent: 0;
 	text-shadow: none;
 	text-transform: none;
 	top: auto;
-	-webkit-transform: none;
-	        transform: none;
-	-webkit-transform-origin: 50% 50% 0;
-	        transform-origin: 50% 50% 0;
-	-webkit-transform-style: flat;
-	        transform-style: flat;
+	transform: none;
+	transform-origin: 50% 50% 0;
+	transform-style: flat;
 	transition: none 0s ease 0s;
 	unicode-bidi: normal;
 	vertical-align: baseline;
diff --git a/test/basic.stage0.expect.css b/test/basic.stage0.expect.css
index f1841fd..01810fb 100644
--- a/test/basic.stage0.expect.css
+++ b/test/basic.stage0.expect.css
@@ -143,21 +143,21 @@ h1.test-custom-selectors,h2.test-custom-selectors,h3.test-custom-selectors,h4.te
 	clear: none;
 	clip: auto;
 	color: #000;
-	-webkit-columns: auto;
-	        columns: auto;
-	-webkit-column-count: auto;
-	        column-count: auto;
-	-webkit-column-fill: balance;
-	        column-fill: balance;
+	-moz-columns: auto;
+	     columns: auto;
+	-moz-column-count: auto;
+	     column-count: auto;
+	-moz-column-fill: balance;
+	     column-fill: balance;
 	grid-column-gap: normal;
-	-webkit-column-gap: normal;
-	        column-gap: normal;
-	-webkit-column-rule: medium none currentColor;
-	        column-rule: medium none currentColor;
-	-webkit-column-span: 1;
-	        column-span: 1;
-	-webkit-column-width: auto;
-	        column-width: auto;
+	-moz-column-gap: normal;
+	     column-gap: normal;
+	-moz-column-rule: medium none currentColor;
+	     column-rule: medium none currentColor;
+	-moz-column-span: 1;
+	     column-span: 1;
+	-moz-column-width: auto;
+	     column-width: auto;
 	content: normal;
 	counter-increment: none;
 	counter-reset: none;
@@ -197,10 +197,8 @@ h1.test-custom-selectors,h2.test-custom-selectors,h3.test-custom-selectors,h4.te
 	page-break-after: auto;
 	page-break-before: auto;
 	page-break-inside: auto;
-	-webkit-perspective: none;
-	        perspective: none;
-	-webkit-perspective-origin: 50% 50%;
-	        perspective-origin: 50% 50%;
+	perspective: none;
+	perspective-origin: 50% 50%;
 	position: static;
 	right: auto;
 	-moz-tab-size: 8;
@@ -208,18 +206,16 @@ h1.test-custom-selectors,h2.test-custom-selectors,h3.test-custom-selectors,h4.te
 	     tab-size: 8;
 	table-layout: auto;
 	text-align: left;
-	text-align-last: auto;
+	-moz-text-align-last: auto;
+	     text-align-last: auto;
 	text-decoration: none;
 	text-indent: 0;
 	text-shadow: none;
 	text-transform: none;
 	top: auto;
-	-webkit-transform: none;
-	        transform: none;
-	-webkit-transform-origin: 50% 50% 0;
-	        transform-origin: 50% 50% 0;
-	-webkit-transform-style: flat;
-	        transform-style: flat;
+	transform: none;
+	transform-origin: 50% 50% 0;
+	transform-style: flat;
 	transition: none 0s ease 0s;
 	unicode-bidi: normal;
 	vertical-align: baseline;
-- 
2.33.1

